contributing/ci-bio-formats.txt

```diff
@@ -165,11 +200,76 @@ breaking jobs are listed under the :jenkinsview:`Breaking` view tab of
 Jenkins.
 
 .. glossary::
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-trigger`
+
+		This job runs the daily Bio-Formats jobs used for reviewing the PRs
+		opened against the develop branch of Bio-Formats by running basic unit
+		tests, checking for open file handles, and checking for regressions
+		across a representative subset of the data repository
+
+		#. Triggers :term:`BIOFORMATS-5.1-breaking-push`
+		#. Triggers :term:`BIOFORMATS-5.1-breaking-build`,
+		   :term:`BIOFORMATS-5.1-breaking-cpp`,
+		   :term:`BIOFORMATS-5.1-breaking-test_images_good` and
+		   :term:`BIOFORMATS-5.1-breaking-test_images_good-win`
+
+		See :jenkinsjob:`the build graph <BIOFORMATS-5.1-breaking-trigger/lastSuccessfulBuild/BuildGraph>`
 
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-push`
+
+		This job merges all the breaking PRs opened against develop
+
+		#. |merge| labeled as breaking
+		#. Pushes the branch to develop/breaking/trigger
+
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-build`
+
+		This job builds the merge artifacts of Bio-Formats
+
+		#. Checks out the develop/breaking/trigger branch of the
+		   snoopycrimecop fork of bioformats.git_
+		#. |buildBF|
+		#. |fulltestBF|
+		#. Triggers :term:`BIOFORMATS-5.1-breaking-matlab`
+
+
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-matlab`
+
+		This job runs the MATLAB tests of Bio-Formats
+
+		#. Collects the MATLAB artifacts and unit tests from
+		   :term:`BIOFORMATS-5.1-breaking-build`
+		#. Runs the MATLAB unit tests under
+		   :file:`components/bio-formats/test/matlab` and collect the results
+
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-cpp`
+
+		This job is used to build the merge native C++ implementation for
+		Bio-Formats
+
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-test_images_good`
+
+		This job runs automated tests against the test_images_good subset on
+		Linux
+
+		#. Checks out the develop/breaking/trigger branch of the
+		   snoopycrimecop fork of bioformats.git_
+		#. Runs automated tests against
+		   :file:`/ome/data_repo/test_images_good`
+
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-test_images_good-win`
+
+		This job runs automated tests against the test_images_good subset on
+		Windows
+
+		#. Checks out the develop/breaking/trigger of the
+		   snoopycrimecop fork of bioformats.git_
+		#. Runs automated tests against
+		   :file:`\\squig.openmicroscopy.org.uk\ome-data-repo\test_images_good`
```

Comment:
As above, slashes around ome-data-repo not rendered on staging and inconsistency in how you refer to this folder